### ISSUE TITLE
Email Flowbird IT about API access

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#6791]
Van send email addresses of Flowbird IT so we can ask questions
- Darren C. 
- Lionel D.

**Comment by dianamartin** [comment]
Emailed contacts
![image.png](https://images.zenhubusercontent.com/5cb4c11373d7e234297de5cf/38c36380-aa44-45ac-882f-b2a389f414da)

**Comment by johnclary** [comment]
Still waiting on access. We're tracking in this in another issue in the epic.
